libraries/AP_HAL_ChibiOS/hwdef/CubeOrange/hwdef-bl.dat

```diff
@@ -56,3 +56,7 @@ PC15 ACCEL_EXT_CS CS
 PD7 BARO_CS CS
 PE4 MPU_EXT_CS CS
 PD10 FRAM_CS CS SPEED_VERYLOW
+
+# disable peripheral and sensor power in the bootloader
```

Comment:
this has the disadvantage that a soft reboot will cause loss of GPS lock, which is really annoying in pre-flight